### PR TITLE
[zh] Update Chinese localization on content/zh/docs/contribute/_index.md  fix issue #23810

### PR DESCRIPTION
**Pull request by zhiguo-lu**
Update content/zh/docs/contribute/_index.md  to the latest English Version

Which issue(s) this PR fixes:
#23810

### CODE REVIEW

**Comment by zhiguo-lu** [comment]
Fixes #23810

File: content/zh/docs/contribute/_index.md [hunk 1]
```diff
@@ -1,9 +1,186 @@
 ---
-title: 贡献新内容
-weight: 20
+content_type: concept
+title: 贡献kubernetes文档
```
Comment (idealhack) [inline content/zh/docs/contribute/_index.md:3]:
```suggestion
title: 贡献 Kubernetes 文档
```

**Review by idealhack — COMMENTED** [review]
thanks! looking good overall, could you fix some small styling issues?

1. there's a space between English and Chinese word.
1. there's _no_ space between two Chinese words.
1. some English words are properly capitalized

File: content/zh/docs/contribute/_index.md (hunk 1 above)
Comment (idealhack) [reply content/zh/docs/contribute/_index.md:3]: this is an example, there're maybe 10 to 20 places like this

Comment (zhiguo-lu) [reply content/zh/docs/contribute/_index.md]: thanks for your kindly comment, it's really my problem and I have fixed it.
by the way, translate work take me 2 hour, while git things take  2 days. :)

File: content/zh/docs/contribute/_index.md [hunk 2]
```diff
@@ -1,9 +1,186 @@
...
+
+- Improve existing content
+- Create new content
+- Translate the documentation
+- Manage and publish the documentation parts of the Kubernetes release cycle
+
+Kubernetes documentation welcomes improvements from all contributors, new and experienced!
+-->
+
+本网站由 [Kubernetes Docs 特别兴趣小组](/zh/docs/contribute/#参与docs特别兴趣小组) 维护.
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:43]: SIG 一词第一次出现时用 `SIG（特别兴趣小组）`，之后不必再翻译。

File: content/zh/docs/contribute/_index.md [hunk 3]
```diff
@@ -1,9 +1,186 @@
...
+
+SIG Docs communicates with different methods:
+
+- [Join `#sig-docs` on the Kubernetes Slack instance](https://slack.k8s.io/). Make sure to
+  introduce yourself!
+- [Join the `kubernetes-sig-docs` mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs),
+  where broader discussions take place and official decisions are recorded.
+- Join the [weekly SIG Docs video meeting](https://github.com/kubernetes/community/tree/master/sig-docs). Meetings are always announced on `#sig-docs` and added to the [Kubernetes community meetings calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles). You'll need to download the [Zoom client](https://zoom.us/download) or dial in using a phone.
+-->
+## 参与 Docs 特别兴趣小组
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:163]:
```suggestion
## 参与 SIG Docs 工作    {#get-involved-with-sig-docs}
```

File: content/zh/docs/contribute/_index.md [hunk 4]
```diff
@@ -1,9 +1,186 @@
...
+
+Some tasks require more trust and more access in the Kubernetes organization.
+See [Participating in SIG Docs](/docs/contribute/participate/) for more details about
+roles and permissions.
 -->
+## 入门
+
+任何人都可以提出文档方面的问题（issue），或贡献一个变更，用拉取请求（PR）的方式提交到
+[GitHub 上的 `kubernetes/website` 仓库](https://github.com/kubernetes/website)。
+当然您需要熟练使用 [git](https://git-scm.com/) 和 [GitHub](https://lab.github.com/) 才能在 Kubernetes 社区中有效工作。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:84]: 文档中不必过于客气地使用`您`.

File: content/zh/docs/contribute/_index.md [hunk 5]
```diff
@@ -1,9 +1,186 @@
...
+
+参与文档编制:
+
+1. 签署 CNCF 的 [贡献者许可协议](https://github.com/kubernetes/community/blob/master/CLA.md)。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:88]:
```suggestion
1. 签署 CNCF 的[贡献者许可协议](https://github.com/kubernetes/community/blob/master/CLA.md)。
```

连接中的内容一样要在中英文间加空格。但如果都是中文，则空白要删除。

Comment (tengqm) [reply content/zh/docs/contribute/_index.md:88]: 下同。

File: content/zh/docs/contribute/_index.md [hunk 6]
```diff
@@ -1,9 +1,186 @@
...
+1. 熟悉 [文档仓库](https://github.com/kubernetes/website)
+   和网站的 [静态站点生成器](https://gohugo.io)。
+1. 确保理解
+   [发起拉取请求 (PR)](/zh/docs/contribute/new-content/open-a-pr/) 和
+   [审查变更](/zh/docs/contribute/review/reviewing-prs/) 的基本流程。
+
+有些任务要求 Kubernetes 组织内更高的信任级别和访问权限。
+阅读 [参与 Docs 特别兴趣小组](/zh/docs/contribute/participate/) ，获取角色和权限的更多细节。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:96]:
```suggestion
阅读[参与 SIG Docs 工作](/zh/docs/contribute/participate/) ，获取角色和权限的更多细节。
```

File: content/zh/docs/contribute/_index.md [hunk 7]
```diff
@@ -1,9 +1,186 @@
...
+- Learn about [page content types](/docs/contribute/style/page-content-types/)
+  and [Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/).
+-->
+## 第一次贡献
+
+- 通读 [贡献概述](/zh/docs/contribute/new-content/overview/)，了解参与贡献的不同方式。
+- 查看 [`kubernetes/website` 问题列表](https://github.com/kubernetes/website/issues/)
+  ，检索最适合作为切入点的问题。
+- 在现有文档上，[使用 GitHub 提交变更 (PR)](/zh/docs/contribute/new-content/open-a-pr/#changes-using-github)，
+  掌握更多的 GitHub 文件操作问题。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:120]:
```suggestion
  掌握在 GitHub 上登记 Issue 的方法。
```

File: content/zh/docs/contribute/_index.md [hunk 8]
```diff
@@ -1,9 +1,186 @@
...
+- Kubernetes 社区其他成员会 [评审拉取请求 (PR)](/zh/docs/contribute/review/reviewing-prs/)， 以确保文档精准和语言流畅。
+- 阅读 kubernetes的 [内容指南](/zh/docs/contribute/style/content-guide/) 和 [风格指南](/docs/contribute/style/style-guide/) ，你才可以发表出有见地的评论。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:122]:
```suggestion
- 阅读 kubernetes 的[内容指南](/zh/docs/contribute/style/content-guide/) 和
  [风格指南](/docs/contribute/style/style-guide/) ，以便发表有见地的评论。
```

File: content/zh/docs/contribute/_index.md [hunk 9]
```diff
@@ -1,9 +1,186 @@
...
+- Document [features in a release](/docs/contribute/new-content/new-features/).
+- Participate in [SIG Docs](/docs/contribute/participate/), and become a
+  [member or reviewer](/docs/contribute/participate/roles-and-responsibilities/).
+                       
+- Start or help with a [localization](/docs/contribute/localization/).
+-->
+## 下一步
+
+- 学习在仓库的 [本地克隆中工作](/zh/docs/contribute/new-content/open-a-pr/#fork-the-repo)
+  。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:140]: 这种标点符号不要放在单独一行，不要放在行首。

File: content/zh/docs/contribute/_index.md [hunk 10]
```diff
@@ -1,9 +1,186 @@
...
+- 记录 [发行版的特性](/zh/docs/contribute/new-content/new-features/)。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:141]:
```suggestion
- 为[发行版的特性](/zh/docs/contribute/new-content/new-features/)编写文档。
```

File: content/zh/docs/contribute/_index.md [hunk 11]
```diff
@@ -1,9 +1,186 @@
...
+- 加入 [Docs 特别兴趣小组](/zh/docs/contribute/participate/), 并成为
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:142]:
```suggestion
- 加入 [SIG Docs](/zh/docs/contribute/participate/), 并成为
```

File: content/zh/docs/contribute/_index.md [hunk 12]
```diff
@@ -1,9 +1,186 @@
...
+  [成员或评审者](/zh/docs/contribute/participate/roles-and-responsibilities/)。
+                       
+- 开始或帮助 [本地化](/zh/docs/contribute/localization/) 工作。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:145]:
```suggestion
- 开始或帮助[本地化](/zh/docs/contribute/localization/)工作。
```

File: content/zh/docs/contribute/_index.md [hunk 13]
```diff
@@ -1,9 +1,186 @@
...
+
+- [Join `#sig-docs` on the Kubernetes Slack instance](https://slack.k8s.io/). Make sure to
+  introduce yourself!
+- [Join the `kubernetes-sig-docs` mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs),
+  where broader discussions take place and official decisions are recorded.
+- Join the [weekly SIG Docs video meeting](https://github.com/kubernetes/community/tree/master/sig-docs). Meetings are always announced on `#sig-docs` and added to the [Kubernetes community meetings calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles). You'll need to download the [Zoom client](https://zoom.us/download) or dial in using a phone.
+-->
+## 参与 Docs 特别兴趣小组
+
+[Docs 特别兴趣小组](/zh/docs/contribute/participate/) 是贡献者们发布、维护 Kubernetes 文档的团体。参与Docs 特别兴趣小组是 Kubernetes 贡献者（开发者和其他人员）对 Kubernetes 项目产生重大影响力的好方式。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:165]:
```suggestion
[SIG Docs](/zh/docs/contribute/participate/) 是负责发布、维护 Kubernetes 文档的贡献者团体。
参与 SIG Docs 是 Kubernetes 贡献者（开发者和其他人员）对 Kubernetes 项目产生重大影响力的好方式。
```

File: content/zh/docs/contribute/_index.md [hunk 14]
```diff
@@ -1,9 +1,186 @@
...
+
+Docs 特别兴趣小组沟通的几种方式:
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:167]:
```suggestion
SIG Docs 的几种沟通方式：
```

File: content/zh/docs/contribute/_index.md [hunk 15]
```diff
@@ -1,9 +1,186 @@
...
+
+- [加入 Kubernetes 在 Slack 上的`#sig-docs` 频道](https://slack.k8s.io/)。
+  一定记得自我介绍!
+- [加入`kubernetes-sig-docs` 邮件列表](https://groups.google.com/forum/#!forum/kubernetes-sig-docs),
+  这里有更广泛的讨论，和官方决策的记录。
+- 参加 [Docs特别兴趣小组的每周视频会议](https://github.com/kubernetes/community/tree/master/sig-docs)。会议总是在 `#sig-docs` 上发布，同时添加到 [Kubernetes社区会议日历](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles)。您需要下载 [Zoom 客户端软件](https://zoom.us/download)，并电话拨号接入。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:173]:
```suggestion
- 参加 [Docs特别兴趣小组的每周视频会议](https://github.com/kubernetes/community/tree/master/sig-docs)。
  会议总是在 `#sig-docs` 上宣告，同时添加到
  [Kubernetes 社区会议日历](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles)。
  你需要下载 [Zoom 客户端软件](https://zoom.us/download)，或电话拨号接入。
```

File: content/zh/docs/contribute/_index.md [hunk 16]
```diff
@@ -1,9 +1,186 @@
...
+
+<!--
+## Other ways to contribute
+
+- Visit the [Kubernetes community site](/community/). Participate on Twitter or Stack Overflow, learn about local Kubernetes meetups and events, and more.
+- Read the [contributor cheatsheet](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) to get involved with Kubernetes feature development.
+- Submit a [blog post or case study](/docs/contribute/new-content/blogs-case-studies/).
+-->
+## 其他贡献的方式
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:182]:
```suggestion
## 其他贡献方式
```

File: content/zh/docs/contribute/_index.md [hunk 17]
```diff
@@ -1,9 +1,186 @@
...
+
+- 访问 [Kubernetes 社区网站](/zh/community/)。 参与 Twitter 或 Stack Overflow, 了解 Kubernetes 当地的聚会和活动, 等等。
+- 阅读 [贡献者备忘单](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) ，参与Kubernetes 功能开发。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:185]:
```suggestion
- 阅读[贡献者备忘单](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet) ，
  参与 Kubernetes 功能开发。
```

File: content/zh/docs/contribute/_index.md [hunk 18]
```diff
@@ -1,9 +1,186 @@
...
+- 提交一篇 [博客文章或案例研究](/zh/docs/contribute/new-content/blogs-case-studies/)。
```
Comment (tengqm) [inline content/zh/docs/contribute/_index.md:186]:
```suggestion
- 提交一篇[博客文章或案例研究](/zh/docs/contribute/new-content/blogs-case-studies/)。
```

**Comment by zhiguo-lu** [comment]
Qiming, Great comment! It's really help.
I had fixed it and check it twice.
thanks a lot!

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Comment by zhiguo-lu** [comment]
fix #23810

**Pull request merged**